File: app/app/src/main/java/com/example/n_ai_tupeu/database/Account.java
package com.example.n_ai_tupeu.database;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "account")
public class Account {
    @PrimaryKey
    @NonNull
    private String id;
    private String username;
    private String password;

    public Account(@NonNull String id, String username, String password) {
        this.id = id;
        this.username = username;
        this.password = password;
    }

    @NonNull
    public String getId() {
        return id;
    }

    public void setId(@NonNull String id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: GameAPI/GameAPI/Models/Account.cs
namespace GameAPI.Models
{
    public class Account
    {
        public Account(string? username, string? password)
        {
            Id= Guid.NewGuid();
            Username = username;
            Password = password;
        }

        public Guid Id { get; set; }
        public string? Username { get; set; }
        public string? Password { get; set; }
    }
}
